### ISSUE TITLE
Error in gym setup command: 'extras_require'

### ISSUE THREAD

**Issue by ethan-alderson** [aidudezzz/deepbots#143]
![image](https://github.com/aidudezzz/deepbots/assets/50850846/b7b3e393-dc0d-4fa3-932f-3c957681c4bd)

Running into this error when attempting pip install deepbots, I have upgraded pip, installed gym as a dependency, and am running Python 3.11.3. Am running into the same issue on other machines when attempting the same install.

**Comment by KelvinYang0320** [comment]
@ethan-alderson Hi, thank you for opening an issue.
Could you share your setuptools version with us and try `pip install setuptools==65.5.0`?
https://github.com/openai/gym/issues/3176#issuecomment-1408598628

**Comment by tsampazk** [comment]
Hey @ethan-alderson, did you have any luck with this issue?

**Comment by ethan-alderson** [comment]
Yep, had some clashing dependencies, worked in a fresh virtual environment!

**Comment by tsampazk** [comment]
I'm reopening this issue and will keep it open as it is recurring and other users might face it until the gym setup stuff is ironed out. Also pinned it.

**Comment by tsampazk** [comment]
> @ethan-alderson Hi, thank you for opening an issue. Could you share your setuptools version with us and try `pip install setuptools==65.5.0`? [openai/gym#3176 (comment)](https://github.com/openai/gym/issues/3176#issuecomment-1408598628)

**For other users that run into this issue**, i investigated further, and apart from setup tools version, [pip version also interferes with gym installation](https://github.com/openai/gym/issues/3176#issuecomment-1560026649). 

If `pip install setuptools==65.5.0` itself doesn't fix it, try:

 `pip install setuptools==65.5.0 pip==21`
 
 This modifies the user's environment unfortunately. We will investigate (#147) into changing our gym dependency to gymnasium (see also [this](https://github.com/Kaggle/kaggle-environments/issues/235)).
